internal/core/operations/remove_rooms/remove_rooms_executor.go

```diff
@@ -71,13 +69,8 @@ func (e *RemoveRoomsExecutor) Execute(ctx context.Context, op *operation.Operati
 		if err != nil {
 			reportDeletionFailedTotal(op.SchedulerName, op.ID)
 			logger.Warn("failed to remove rooms", zap.Error(err))
-			deleteErr := fmt.Errorf("failed to remove room: %w", err)
 
-			if errors.Is(err, serviceerrors.ErrGameRoomStatusWaitingTimeout) {
-				return operations.NewErrTerminatingPingTimeout(deleteErr)
-			}
-
-			return operations.NewErrUnexpected(fmt.Errorf("failed to remove room by ids: %w", err))
```

Comment:
This line is useless since DeleteRoom method doesn't return ErrGameRoomStatusWaitingTimeout error anymore